### ISSUE TITLE
Tool icons

### ISSUE THREAD

**Issue by Granada1** [TheGameCreators/GameGuruRepo#1794]
Hi guys, it would be cool if we could move these across to make life simpler

![ATrees_005](https://user-images.githubusercontent.com/35897659/150012048-8b48b6b5-973d-409e-9562-70ba69f71369.png)

**Comment by rvanner** [comment]
@Granada1 We have no plans to move these icons at this time.

**Comment by Necrym59** [comment]
I have to agree all  - the object and terrain tools should be moved with a sperator between the other group
